utilities/zookeeper/R/corrector-aardvark.R

```diff
@@ -44,6 +44,52 @@ default_state_params <- function(
   )
 }
 
+# 
+special_flags_marker <- function(
```

Comment:
If everything is `NA`, this will output a tibble with 1 row of NA's right? We need to do a few things here. 
1. Throw an error if pairs don't match up.
2. It's better to return an empty tibble (with 0 rows) by default.

Reply:
And see the comment below about documentation.